Find a functionally equivalent code:android.provider.Settings.Secure.putIntForUser:COMMENT
Method Modifier: public      static      hidden      
Comment:/**
 * @hide
 */

@@ -1,7 +1,3 @@
 {
-    if (LOCATION_MODE.equals(name)) {
-        // TODO: once b/10491283 fixed, remove this hack
-        return setLocationModeForUser(cr, value, userHandle);
-    }
     return putStringForUser(cr, name, Integer.toString(value), userHandle);
 }

